main.go

```diff
@@ -72,6 +72,11 @@ func main() {
 	// nolint: typecheck
 	rend := render.NewWithDefaultClient(assets.Asset, assets.AssetNames, cfg.PatternLibraryAssetsPath, cfg.SiteDomain)
 
+	cfg.FeedbackAPI = &config.FeedbackConfig{
```

Comment:
I imagine this would be to set the default values.
The default values should be set in config.go (otherwise here we would override the values, even if they were obtained from a secrets file).

Could you move this to the default config, please? (config.go line 65)
```
cfg := &Config{
...
   FeedbackAPI: &FeedbackConfig {
      ...
   }
...
}
```

Reply:
Didnt see this message until now. Will make the change.